internal/infrastructure/kubernetes/deployment.go

```diff
@@ -401,7 +401,7 @@ func expectedRateLimitContainers(infra *ir.RateLimitInfra) []corev1.Container {
 				},
 				{
 					Name:  "LOG_LEVEL",
-					Value: "info",
+					Value: "debug",
```

Comment:
Why is this log level now debug?

Reply:
adds verbosity for this new feature, which can help triaging future issues, can move it to `info` if that is the preference for default

Reply:
reverted to `info`

Reply:
Thanks for reverting to info. I think it makes sense to have a consistent logging level across EG components.